Source/Scene/Cesium3DTileset.js

```diff
@@ -788,19 +816,57 @@ define([
         }
     }
 
-    function getScreenSpaceError(geometricError, tile, frameState) {
+    var scratchPositionNormal = new Cartesian3();
+
+    function updateDynamicScreenSpaceError(tileset, frameState) {
+        var camera = frameState.camera;
+        var positionCartographic = camera.positionCartographic;
+        var height = positionCartographic.height;
+
+        // The range where the density starts to lessen
```

Comment:
Yeah that way could work for most cases, we may need to query the terrain height in order to get accurate results for tilesets that are at higher elevations.

The max height is used as an approximate way to see how far the camera is from the tileset. When the camera is high enough I don't apply any dynamic sse because it tends to make the whole tileset pretty blurry. My calculations are really meant to apply dynamic SSE at street level views where a wide range of LODs are visible at the same time. At the moment height seemed to be the best way to measure this.